app/views/accessibility_statement.scala.html

```diff
@@ -94,8 +96,8 @@ <h2 class="govuk-heading-l">@messages("compliant.nine.heading")</h2>
     <p class="govuk-body">@messages("compliant.nine.p1")</p>
 
     <p class="govuk-body">@messages("compliant.nine.p2.1")
-        <a href="http://www.digitalaccessibilitycentre.org/" class="govuk-link">@messages("compliant.nine.p2.2")</a>.
-        @messages("compliant.nine.p2.3")
+         <a href="http://www.digitalaccessibilitycentre.org/" class="govuk-link">@messages("compliant.nine.p2.2")</a>
```

Comment:
The resulting statement has a space between end of link text and the full stop - as below

> The service was built using parts that were tested by the Digital Accessibility Centre . The full service was tested by HMRC and included disabled users.